src/wizard/javascript/angular/with-error-monitoring-and-performance.md

```diff
@@ -0,0 +1,105 @@
+---
+name: Angular
+doc_link: https://docs.sentry.io/platforms/javascript/guides/angular/
+support_level: production
+type: framework
+---
+
+## Install
+To use Sentry with your Angular application, you'll need `@sentry/angular-ivy` or `@sentry/angular`, Sentry’s Browser Angular SDKs:
+
+- If you're using Angular 12 or newer, use `@sentry/angular-ivy`
+- If you're using Angular 10 or 11, use `@sentry/angular`
+
+Add the Sentry SDK as a dependency using `yarn` or `npm`:
+
+```bash
+# Using yarn (Angular 12+)
+yarn add @sentry/angular-ivy
+# Using yarn (Angular 10 and 11)
+yarn add @sentry/angular
+
+# Using npm (Angular 12+)
+npm install --save @sentry/angular-ivy
+# Using npm (Angular 10 and 11)
+npm install --save @sentry/angular
+```
+
+## Configure
+You should `init` the Sentry browser SDK in your `main.ts` file as soon as possible during application load up, before initializing Angular:
+
+```javascript
+import { enableProdMode } from "@angular/core";
+import { platformBrowserDynamic } from "@angular/platform-browser-dynamic";
+// import * as Sentry from "@sentry/angular" // for Angular 10/11 instead
+import * as Sentry from "@sentry/angular-ivy";
+
+import { AppModule } from "./app/app.module";
+
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+  integrations: [
+    new Sentry.BrowserTracing({
+      tracePropagationTargets: ["localhost", "my-site-url.com", /^\//],
+      routingInstrumentation: Sentry.routingInstrumentation,
+    }),
+  ],
+  // Performance Monitoring
+  tracesSampleRate: 1.0, // Capture 100% of the transactions, reduce in production!
+});
+
+enableProdMode();
+platformBrowserDynamic()
+  .bootstrapModule(AppModule)
+  .then(success => console.log(`Bootstrap success`))
+  .catch(err => console.error(err));
+```
+
+### ErrorHandler and Tracer
+
+The Sentry Angular SDK exports a function to instantiate `ErrorHandler` provider that will automatically send JavaScript errors captured by the Angular's error handler.
+
+```javascript
+import { APP_INITIALIZER, ErrorHandler, NgModule } from "@angular/core";
+import { Router } from "@angular/router";
+// import * as Sentry from "@sentry/angular" // for Angular 10/11 instead
+import * as Sentry from "@sentry/angular-ivy";
+
+@NgModule({
+  // ...
+  providers: [
+    {
+      provide: ErrorHandler,
+      useValue: Sentry.createErrorHandler({
+        showDialog: true,
+      }),
+    },
+    {
+      provide: Sentry.TraceService,
+      deps: [Router],
+    },
+    {
+      provide: APP_INITIALIZER,
+      useFactory: () => () => {},
+      deps: [Sentry.TraceService],
+      multi: true,
+    },
+  ],
+  // ...
+})
+export class AppModule {}
```

Comment:
I just stumbled across this. This is definitely required to set up the SDK properly in Angular apps. Otherwise the routing instrumentation doesn't work  (==> no/incorrect pageload navigation transactions) and errors would only be caught by the global handlers.